service/resource/cloudformationv2/adapter/adapter.go

```diff
@@ -127,3 +134,23 @@ func NewHostPost(cfg Config) (Adapter, error) {
 
 	return a, nil
 }
+
+// getImageID returns the EC2 AMI for the configured region.
+func getImageID(customObject v1alpha1.AWSConfig) (string, error) {
+	region := keyv2.Region(customObject)
+
+	// Container Linux AMIs for each active region.
+	// NOTE: AMIs should always be for HVM virtualisation and not PV.
+	imageIDs := map[string]string{
+		"eu-central-1": "ami-90c152ff",
+		"eu-west-1":    "ami-32d1474b",
+		"us-west-2":    "ami-dc4ce6a4",
+	}
```

Comment:
Here we need to specify the AMIs. I think it makes sense to have this in the main adapter as this is where we set these values.

I think we should only specify values for active regions. The error message is clear and we can add more as we expand to more regions. WDYT?

Reply:
Should we add here in the comment the CoreOS version and link to https://coreos.com/os/docs/latest/booting-on-ec2.html ? This way it is easier to know in the code what those ami's stand for (without having to look into the version bundle) and its also easy to verify.

Alternatively we could also just link directly to the versioned link like following:
https://coreos.com/os/docs/1590.2.0/booting-on-ec2.html

Just some ideas as "documentation in code"

Reply:
Any PR for changing `ami`s will look like we are changing some logic or so, because of file being updated :/
But still don't see any better place for that